src/objects.js

```diff
@@ -5788,6 +5788,24 @@ SpriteMorph.prototype.allMessageNames = function () {
     return msgs;
 };
 
+SpriteMorph.prototype.allSendersOf = function (message) {
+    if (typeof message === 'number') {message = message.toString(); }
+    return this.scripts.children.filter(morph => {
+        var event;
+        if (morph.selector) {
+            if (
+                contains(['doBroadcast', 'doBroadcastAndWait'], morph.selector)
```

Comment:
also here, I think it's probably a good idea to also include the new 'doSend' selector

Reply:
Also adding it now.